### PR TITLE
Feature/ptwo 217 exmg sortable

### PR DESCRIPTION
**Pull request by markoffk**

### CODE REVIEW

**Comment by markoffk** [comment]
There is still issue with Firefox. So PR should not be merged yet

**Review by marksmits — APPROVED** [review]

File: exmg-sortable.ts [hunk 1]
```diff
@@ -0,0 +1,395 @@
...
+          sourceIndex,
+          targetIndex,
+        },
+      }));
+    }
+
+    this.reset();
+
+    this.dragRequestPending = false;
+    this.style.webkitUserSelect = 'text';
```
Comment (pgoc) [inline exmg-sortable.ts]: You only updated webkitUserSelect what about other browsers ? Shouldn't you also update userSelect property ?

File: exmg-sortable.ts [hunk 2]
```diff
@@ -0,0 +1,395 @@
...
+        const index = this.animatedElements.indexOf(node);
+        Object.assign(node.style, {
+          willChange: 'initial',
+        });
+        if (index !== -1) {
+          // splice out when done to unlock as a valid target
+          this.animatedElements.splice(index, 1);
+        }
+
+        resolve();
```
Comment (pgoc) [inline exmg-sortable.ts:302]: resolved value is void - it means that this.animationPromise should be type Promise<void> instead Promise<any>

**Review by pgoc — APPROVED** [review]

File: exmg-sortable.ts (hunk 2 above)
Comment (markoffk) [reply exmg-sortable.ts:302]: done

File: exmg-sortable.ts (hunk 1 above)
Comment (markoffk) [reply exmg-sortable.ts]: done

**Pull request merged**